src/server.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express';
+import http from 'http';
+import SocketIO from 'socket.io';
+
+const addEventListeners = (socket) => {
```

Comment:
Arrow function with a single parameter doesn't need parenthesis

```javascript
const addEventListeners = socket => {}
```

---
How about ```addEventListener```?

Reply:
You're right. It's my mistake.

Reply:
You're right. It's my mistake.

Reply:
Eslint is not es6's coding standard but providing us to configure code styles.

Your reference indicates that it can be discouraged.

So, we can choose one encourage or discourage this rule.

In (airbnb's coding standards)[https://github.com/airbnb/javascript/blob/master/README.md#arrow-functions], they say that a single argument can stand alone without braces.

Reply:
In this time, It's my mistake.
We are following airbnb's coding standards, I will edit this line